src/core/net/nd6.hpp

```diff
@@ -525,6 +532,67 @@ class RaFlagsExtOption : public Option, private Clearable<RaFlagsExtOption>
 
 static_assert(sizeof(RaFlagsExtOption) == 8, "invalid RaFlagsExtOption structure");
 
+class TxMessage
```

Comment:
All the top-level classes in nd6.hpp have a description (comment) header, but for this one it's missing? I think it should be added.